### ISSUE TITLE
test components

### ISSUE THREAD

**Issue by r-tt** [tomitribe/mykola#11]
seems we dont have any test for any components, we should ensure we have the expected cases tested to prevent regression before we move forward on new components.

chai+protractor sounds good but any other alternative enabling to test e2e is good

**Comment by Dexmaster** [comment]
yep, good idea, but for another week it will be additional and not main priority.
